Added Potentiometer Test Code

thesprintingturtle opened the pull request:
Added AnalogInput potentiometer to ArmSub. Added a number to the SmartDashboard to display its value.

Commit:
Added Potentiometer Test Code

Added AnalogInput potentiometer to ArmSub. Added a number to the SmartDashboard to display its value.

coderkevin commented on 2019RobotCode/src/main/java/frc/robot/Subsystems/ArmSub.java:
```diff
@@ -16,9 +17,13 @@
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
 
+  public AnalogInput potentiometer = new AnalogInput(0);
```
This is okay for now, but I'd like to make this private and only expose the angle represented by the value.

Commit:
Made Potentiometer Private

Changed potentiometer in ArmSub to private and added getPotentiometerAngle, which converts the potentiometer value to an angle and returns it. This is currently uncalibrated, and is based off of a 270 degree max rotation.
